app/components/MainSidebar.tsx

```diff
@@ -53,159 +54,84 @@ interface MainSidebarProps {
   }>[];
 }
 
-export const MainSidebar = ({ band, memberRole, bands }: MainSidebarProps) => {
+export const MainSidebar = ({ band, memberRole, bands, openState }: MainSidebarProps) => {
   const user = useUser();
   const [showCreateMenu, setShowCreateMenu] = useState(false);
   const [isPopupOpen, setIsPopupOpen] = useState(false);
   const [isUserOpen, setIsUserOpen] = useState(false);
-  const [isOpen, setIsOpen] = useState(false);
-  const state = isOpen ? "open" : "closed";
-  const { pathname } = useLocation();
-  const fullConfig = resolveConfig(tailwindConfig);
-  const screens = fullConfig.theme?.screens as
-    | Record<string, string>
-    | undefined;
-  const mediumBreakpoint = parseInt(screens?.md || "768px");
 
-  useEffect(() => {
-    const handleIsOpen = () => setIsOpen(window.innerWidth > mediumBreakpoint);
-    window.addEventListener("resize", handleIsOpen);
-    handleIsOpen();
-    return () => window.removeEventListener("resize", handleIsOpen);
-  }, [mediumBreakpoint]);
+  const { pathname } = useLocation();
 
   const isActive = pathname.split("/")[2].includes("user");
 
+  const fetcher = useFetcher()
+
   return (
     <AnimatePresence initial={false}>
       <motion.aside
-        animate={state}
+        animate={openState}
         variants={{
           open: { width: "100%" },
           closed: { width: 80 },
         }}
-        className="h-full bg-base-100 w-full max-w-xs flex shadow-lg z-10 overflow-hidden"
+        className="h-full bg-base-100 w-full max-w-xs flex flex-col shadow-lg z-10"
       >
-        <MaxHeightContainer
```

Comment:
removed max height container because overflow-auto was getting in the way of collapse button hanging outside container